ui.frontend/src/main/webpack/site/js/components/searchForm.ts

```diff
@@ -37,83 +37,106 @@ export const triggerSearch = async (
   }
   const searchInputElementCopy = searchInputElement
 
-  searchInputElementCopy.dataset.loading = 'true'
+  const hasSearchTabs = searchTabs && searchTabs.length > 0;
+
+  if (hasSearchTabs) {
+    searchInputElementCopy.dataset.loading = 'true'
+  }
 
   const searchValue = searchInputElement.value
-  const { onSearch, onSwitchTab, onSearchItemClick, onLoadMoreButtonClick } =
-    options || {}
+  const {onSearch, onSwitchTab, onSearchItemClick, onLoadMoreButtonClick} =
+  options || {}
 
   onSearch?.(searchValue)
 
   if (searchUrl) {
```

Comment:
The two `if` conditions shouldn't be nested:
`if(searchUrl && searchUrl != window.location.pathname)`